hs_composite_resource/models.py

```diff
@@ -440,65 +769,138 @@ def supports_rename_path(self, src_full_path, tgt_full_path):
             src_file_dir = src_full_path
 
         if src_ext and tgt_ext:
-            is_renaming_file = True
+            if src_file_name != tgt_file_name:
+                is_renaming_file = True
```

Comment:
I'm betting renaming and moving are similar enough that we don't need to handle the situation differently.  I think just using the logic around moving a file should cover all requirements of renaming.